GoogleSignIn/Tests/Unit/GIDFakeFetcherService.m

```diff
@@ -63,6 +63,15 @@ - (NSURLSession *)sessionForFetcherCreation {
   return nil;
 }
 
+// TODO: replace with GTMSessionFetcherSessionCreationBlock once that has become
+// available by updating the version dependencies.
+typedef NSURLSession *_Nullable (^GIDFakeFetcherSessionCreationBlock)(
+    id<NSURLSessionDelegate> _Nullable delegate);
+- (NSURLSession *)sessionWithCreationBlock:
```

Comment:
Might add comments to the effect of "this is for pre-release support" since we don't typically build against your HEAD.